src/main/java/org/jenkinsci/plugins/authorizeproject/AuthorizeProjectStrategyDescriptor.java

```diff
@@ -114,4 +114,12 @@ public void configureFromGlobalSecurity(StaplerRequest req, JSONObject js) throw
     public boolean isEnabledByDefault() {
         return true;
     }
+    
+    /**
+     * @return whether configurable for {@link GlobalQueueItemAuthenticator}
+     * @since 1.1.1

```

Comment:
I plan the next relase is 1.2.0.
I'll create another pull request to replace `since 1.1.1` to `since 1.2.0`.